vsg/__main__.py

```diff
@@ -244,6 +286,12 @@ def display_rule_configuration(commandLineArguments, configuration):
         sys.exit(fExitStatus)
 
 
+def validate_files_exist_to_analyze(commandLineArguments):
```

Comment:
Again, I am not sure if passing `commandLineArguments` is good idea. The actual argument is filename.

Reply:
I agree.

Reply:
Update has been pushed.